packages/fluentui/docs/src/components/Playground/renderConfig.ts

```diff
@@ -67,6 +69,16 @@ export const imports: Record<string, CodeSandboxImport> = {
     module: Classnames,
     required: false,
   },
+  formik: {
```

Comment:
Why do you need this?
Do you expect `formik` to be used in anywhere in component example?

Reply:
We are using formik in the prototype, not adding it here was throwing errors

Reply:
removed